metricshub-doc/src/site/markdown/usecases/ping.md

```diff
@@ -0,0 +1,60 @@
+keywords: ping
+description: How to ping resources with MetricsHub
+
+# Pinging resources
+
+## Overview
+
+With **MetricsHub**, you can ping your resources through ICMP (Internet Control Message Protocol) to verify that they can be reached within an acceptable response time. To achieve this, you only need to configure in the `config/metricshub.yaml` file:
+* the resources to be monitored 
+* the `ping` protocol for each of your resource.
+
+Once the monitoring is in place, **MetricsHub** will push the following metrics for each resource (host):
+
+* `metricshub.host.up{protocol="ping"}` (1 for successful ping, 0 for no response)
+* `metricshub.host.up.response_time{protocol="ping"}` (response time in seconds)
+
+  > Note: In Prometheus, these metrics will be renamed `metricshub_host_up` and `metricshub_host_up_response_time_seconds` respectively, to align with Prometheus naming conventions.
+
+In the example below, we ping 11 resources using the **ICMP protocol**. A timeout of 3 seconds is configured. The status (OK, KO) and response times are displayed in a Grafana Dashboard.
+
+![MetricsHub - Pinging resources](../images/metricshub-ping-check-feature.png)
+
+## Procedure
+
+To configure the **Ping Check** feature: 
+
+1. In the `config/metricshub.yaml` file, we configure the 11 resources as follows:
+
+    ```yaml
+        attributes:
```

Comment:
Remove this `attributes: site: bay4` section. It's not needed in our use case and it raises questions.